demisto_sdk/tests/test_files/content_slim/Packs/Sample01/CONTRIBUTORS.md

```diff
@@ -0,0 +1,7 @@
+#### This pack was co-authored by:
+ - Contributor1
+ - Contributor2
+
+Thank you for contributing to Cortex XSOAR.
+
+If you are interested in contributing, visit our contribution process [here](https://xsoar.pan.dev/docs/contributing/contributing).
```

Comment:
do we want these lines and the title be modifiable by the contributor, or maybe we should concatenate them in `mention_contributors_in_readme()`?
@ShahafBenYakir wdyt?

Reply:
We considered this option, but prefer doing this like it is now. 
We want to see this markdown file at Pack folder, with the messages.